cfunits/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: cfunits
+  version: {{environ['CBUILD_CFUNITS_VERSION']}}
+
+source:
+  git_url: https://bekozi@bitbucket.org/cfpython/cfunits-python.git
```

Comment:
I am an ignorant on bitbucket, but is that your fork or the canonical source for cfunits-python?

We also try to be consistent with the original Software name. In this case seems to be `cfunits-python`, correct?